packages/commonwealth/client/scripts/stores/interfaces.ts

```diff
@@ -16,3 +16,7 @@ export interface IHasId {
 export interface IHasAddress {
   address: string;
 }
+
+export interface IHasChainId {
```

Comment:
Can we make this extend `IHasId`?

Reply:
We cannot, unfortunately! I wanted to, but we use `identifiers` for many of our tables.